.ci/update-github.sh

```diff
@@ -24,7 +24,13 @@ if [ "$TARGET_VERSION" != "$CURRENT_VERSION" ]; then
   exit 1;
 fi
 
-./.ci/update-github-milestones.sh "$TARGET_VERSION"
+git checkout checkstyle-"$TARGET_VERSION"
+
+echo "Generating uber jar ...(no clean to keep site resources just in case)"
+mvn -e --no-transfer-progress -Passembly package
+
+echo "Updating GitHub Milestone to $TARGET_VERSION"
+./.ci/update-github-milestone.sh "$TARGET_VERSION"
```

Comment:
No need to update milestone.

It is script for action that happens before release process

Reply:
Changes Done